test/e2e/testdata/istio/istio-issuer.yaml

```
@@ -0,0 +1,8 @@
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: istio-ca
+  namespace: istio-system
+spec:
+  ca:
+    secretName: root-secret
```

```suggestion
    secretName: istio-ca
```

If this Issuer intends to use the self-signed CA cert from the other `istio-certificate.yaml`, we need to make the `secretName` reflect correctly.

---

The issuer  will be using the certificate signed by the cluster-issuer.